atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -290,7 +291,8 @@ public String toString() {
      *       while still keeping a minimum number of idle connections around for fast borrows.
      */
     private GenericObjectPool<CassandraClient> createClientPool() {
-        CassandraClientFactory cassandraClientFactory = new CassandraClientFactory(metricsManager, proxy, config);
+        CassandraClientConfig clientConfig = CassandraClientConfig.of(config);
```

Comment:
It's the only place that uses the clientconfig, and is only called once in the constructor, so didn't feel the need to pull out as a field